model/project_ref.go

```diff
@@ -1022,6 +1023,28 @@ func getCommonProjectVariables(projectIds []string) (*ProjectVars, error) {
 	}, nil
 }
 
+func FindTaskWithinTimePeriod(startedAfter, finishedBefore time.Time,
```

Comment:
I don't this really makes sense to be in projectRef, since it doesn't have anything to do with it besides getting the right ID. Since this is only used by projectTaskGetHandler, this can just be a helper method for that handler.